wondrous-app/components/Common/ProgressBar/styles.tsx

```diff
@@ -33,5 +33,4 @@ export const ProgressBarValue = styled.div`
     props.color ||
     `linear-gradient(269.92deg, ${palette.white} -20.02%, ${palette.highlightBlue} -1.88%, ${palette.violet90} 57.38%) `};
   border-radius: inherit;
-  ${({ color }) => color && `box-shadow: 2px 0 4px ${color}`}
```

Comment:
Is there a reason for removing this?

Reply:
yes, basically the new design has a different design for the progress bar, so I removed this one for them to be alligned